cadl-tests/src/main/java/com/dictionary/BooleanValueAsyncClient.java

```diff
@@ -83,14 +91,14 @@ public Mono<Response<Void>> putWithResponse(BinaryData body, RequestOptions requ
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return A {@link Mono} that completes when a successful response is received.
+     * @return the response body on successful completion of {@link Mono}.
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Void> get() {
+    public Mono<Map<String, Boolean>> get() {
         // Generated convenience method for getWithResponse
         RequestOptions requestOptions = new RequestOptions();
-        return getWithResponse(requestOptions).then();
+        return getWithResponse(requestOptions).map(Response::getValue);
```

Comment:
this likely not compile pass.

you need to use the latest main.

Reply:
ok